includes/class-indieauth-local-authorize.php

```diff
@@ -35,6 +35,7 @@ public function verify_access_token( $token ) {
 			return $return;
 		}
 		$return['last_accessed'] = time();
+		$return['last_ip']       = $_SERVER['REMOTE_ADDR'];
```

Comment:
Shouldn't this be a little more robust?  and do something like..

```php
function getUserIpAddr(){
  if(!empty($_SERVER['HTTP_CLIENT_IP'])) {
    $ip = $_SERVER['HTTP_CLIENT_IP'];
  }  elseif(!empty($_SERVER['HTTP_X_FORWARDED_FOR'])) {
    $ip = $_SERVER['HTTP_X_FORWARDED_FOR'];
  } else{
    $ip = $_SERVER['REMOTE_ADDR'];
  }
  return $ip;
}
```

Reply:
I added it to match the fact that the new core Application Passwords code stores this. I was trying to bring this implementation closer to that for future coding considerations. Same reason I added a uuid. It only uses REMOTE_ADDR. As does the comment implementation that stores IP.

https://github.com/WordPress/WordPress/blob/1ead0719b7897632d4d44309dbd904d9eb96dd28/wp-includes/class-wp-application-passwords.php#L238